x/staking/keeper/querier.go

```diff
@@ -1,4 +1,4 @@
-package querier
+package keeper
```

Comment:
I'm curious why you decided to move the querier to the keeper folder (although I think it is a good choice)

Reply:
querier is required to be in both the keeper and the types folder.. keeper folder for the high level functionality, types for the keeper types